jenkins/bootstrap.py

```diff
@@ -239,7 +241,8 @@ def append_result(gsutil, path, build, version, passed):
         else:
             cache = []
         cache.append({
-            'version': version,
+            'version': version,  # TODO(fejta): retire
```

Comment:
are we just changing this to be consistent? I think the job cache is only used by the release infra right now.